### ISSUE TITLE
Datepicker not applying IDs to inputs - fails a11y

### ISSUE THREAD

**Issue by jason-capsule42** [AlaskaAirlines/auro-datepicker#122]
## Describe the issue

Auro-Datepicker does not use ID attribute values on the auro-input elements. This causes the a11y test to fail.
